### PR TITLE
Update of the omniauth gem version

### PR DESCRIPTION
**Pull request by juliansomoza**
For the integration of filters in LDAP auth querys.

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -32,7 +32,8 @@ def register_middleware(omniauth)
           base: SiteSetting.ldap_base,
           uid: SiteSetting.ldap_uid,
           bind_dn: SiteSetting.ldap_bind_db,
-          password: SiteSetting.ldap_password
+          password: SiteSetting.ldap_password,
+          filter: SiteSetting.ldap_filter,
```
Comment (jonmbake) [inline plugin.rb]: Looks like there is an extra comma here.

File: config/settings.yml [hunk 2]
```diff
@@ -26,3 +26,6 @@ plugins:
   ldap_password:
     client: false
     default: ''
+  ldap_filter:
+    client: false
+    default: '(&(sAMAccountName=%{username})(memberof=CN=group-example,DC=example,DC=com))'
```
Comment (jonmbake) [inline config/settings.yml]: Either `ldap_uid` or `ldap_filter` are required, right? Isn't having a default `ldap_filter` going to make `ldap_uid` irrelevant? We can add this but maybe it should not have a default value ie default to nil.

File: config/locales/server.es.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    ldap_enabled: "Está activado el plugin de LDAP?"
+    ldap_user_create_mode: "Modo de creación de usuario: auto, list o none"
+    ldap_hostname: "Hostname del servidor LDAP"
+    ldap_port: "Puerto del servidor LDAP"
+    ldap_method: "Método de conexión: ssl, tls o plain"
+    ldap_base: "LDAP base"
+    ldap_uid: "LDAP uid"
+    ldap_bind_db: "LDAP bind db (necesario si el servidor LDAP no permite bind anónimo)"
+    ldap_password:  "LDAP bind password (necesario si el servidor LDAP no permite binding anónimo)"
+    ldap_filter:  "Filtro LDAP (para especificar permisos por grupos, etc)"
```
Comment (jonmbake) [inline config/locales/server.es.yml]: Thanks for adding this!

File: config/settings.yml (hunk 2 above)
Comment (juliansomoza) [inline config/settings.yml]: I'm agree with you. This will confuse the users... I removed the default example.

File: plugin.rb (hunk 1 above)
Comment (juliansomoza) [inline plugin.rb]: You're right... Changed.

File: config/locales/server.es.yml (hunk 3 above)
Comment (juliansomoza) [inline config/locales/server.es.yml]: No problem, I will add Italian too.

**Pull request merged**